### ISSUE TITLE
Alt text description doesn't have same size as other in sidebar. 

### ISSUE THREAD

**Issue by iFlameing** [kitconcept/volto-light-theme#217]

**Comment by davisagli** [comment]
@iFlameing A screenshot would be really helpful when reporting issues like this

**Comment by steffenri** [comment]
@iFlameing Could you please create those tickets/issues in gitlab?

**Comment by davisagli** [comment]
Fixed in #218
